src/main/java/com/google/sps/servlets/GmailServlet.java

```diff
@@ -27,7 +27,10 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Serves selected information from the User's Gmail Account */
+/**
+ * Serves selected information from the User's Gmail Account. TODO: Create Servlet Utility to handle
+ * common functions (Issue #26)
```

Comment:
Good job leaving a TODO trail to help find where refactoring needs to be made!

Reply:
Thank you!

Reply:
Thanks!